requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+-r requirements.txt
+mock
+pep8==1.6.2
+pytest
+pytest-cov
+pytest-pep8
```

Comment:
Ah, wasn't aware of that. Sorry, I'm not quite sure if they should be merged then -- maybe there are users who want to render the documentation, but who are not developers? But I'm fine with either.